### ISSUE TITLE
DOC: Instructions for proton are missing

### ISSUE THREAD

**Issue by Kreyren** [PhoenicisOrg/phoenicis#1906]
Instructions to use proton in script.js are missing on https://phoenicisorg.github.io/scripts/Develop/script-js/#custom-script

**Comment by plata** [comment]
This is not true. Proton is a distribution like every other Wine distribution. It doesn't make sense to list every possible distribution in the documentation. Also, this is not even possible because we cannot know which Wine versions json the users are using.

**Comment by Kreyren** [comment]
@plata  
> It doesn't make sense to list every possible distribution in the documentation. 

> Also, this is not even possible because we cannot know which Wine versions json the users are using.

How else are contributors to scripts supposed to write them then and avoid syntax errors?

**Comment by madoar** [comment]
I'm not sure this information should go in the Phoenicis documentation, but I agree with @Kreyren that we should write somewhere about which wine distributions we **officially** support.

**Comment by plata** [comment]
You can see that in Phoenicis... If we write it to a separate document, the only thing that will happen is that the document and the supported distributions diverge.

**Comment by madoar** [comment]
I think we should add a table somewhere containing for all supported wine distributions:
- the name of the wine distribution
- how to use it in a script

**Comment by plata** [comment]
If you want, you can add a new page in the documentation and link it e.g. in the place @Kreyren mentioned.

**Comment by Kreyren** [comment]
@madoar Provide ETA else i will contrib.

**Comment by plata** [comment]
@Kreyren I don't think @madoar is actively working on this currently, so feel free to provide the PR.
